examples/places-service-with-element/app.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {FunctionComponent} from 'react';
+import {GoogleMapsProvider} from '@ubilabs/google-maps-react-hooks';
+
+import PlacesServiceElement from './components/places-service-with-element/places-service-with-element';
+
+import {GOOGLE_MAPS_API_KEY} from '../constants';
+
+import './main.module.css';
+
+const mapOptions = {
+  center: {lat: 53.5582447, lng: 9.647645},
+  zoom: 20,
+  disableDefaultUI: true,
+  zoomControl: false,
+  clickableIcons: false
+};
+
+const App: FunctionComponent<Record<string, unknown>> = () => (
+  <GoogleMapsProvider
+    googleMapsAPIKey={GOOGLE_MAPS_API_KEY}
+    mapOptions={mapOptions}
```

Comment:
This is an example without map, isn't it?

```suggestion
const App: FunctionComponent<Record<string, unknown>> = () => (
  <GoogleMapsProvider
    googleMapsAPIKey={GOOGLE_MAPS_API_KEY}
```